recipe/meta.yaml

```diff
@@ -32,6 +32,8 @@ requirements:
     - python
     - libxcrypt  # [linux and py<39]
     - pybind11
+    - lhapdf >=6.2
+    - zlib >=1.0.0
```

Comment:
You shouldn't put a version constraint here on zlib. It's included in the global pinning: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/9e690262e1bb221e8fc8b370232a0aaf7ef1eb2b/recipe/conda_build_config.yaml#L906-L907

When you rerender the versions are picked from there and included in the `.ci_support` files. The reason for defining it there is that if there is a zlib 2 with a different ABI, a "migration" can be ran to rebuild all dependent packages in order with the new zlib. You can track current migrations on the status page: https://conda-forge.org/status/#migrations

It could also make sense to add pythia and lhapdf to the global pinning but I'll leave that up to you.

Reply:
Anytime there is a version that is deemed ABI incompatible (as defined by the run_exports pattern of lhapdf) a PR like https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6215 will be opened.

If you want to do the migration, you can ask someone from core to merge it (including me) which will prompt the bots to issue PRs to dependent packages in dependency order.

When all of them are done, a PR like https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6275 is opened to update the version in the global pinning file.